modules/dnn/src/layers/einsum_layer.cpp

```diff
@@ -143,25 +140,18 @@ Mat batchwiseMatMul(
 };
 
 Mat Transpose(
-    const cv::Mat& input,
+    const Mat& input,
```

Comment:
Are these functions, such as `batchwiseMatMul`, `Transpose`, defined and declared only in this file? If so, I propose to add `static`.

Reply:
done